frappe/desk/query_report.py

```diff
@@ -174,6 +175,18 @@ def get_script(report_name):
 	}
 
 
+def prepare_filter(filter):
+    filter = {
+        "fieldname": filter.fieldname,
+        "label": filter.label,
+        "fieldtype": filter.fieldtype,
+        "width": "80",
+        "options": filter.options,
+        "reqd": 1
+    }
+    return filter
```

Comment:
Lets not prepare filter just return `report.filters` directly along with script, html_format & execution_time
and we can handle the filter from client side

Reply:
@shariquerik - I made the necessary changes as per your comments.